### PR TITLE
feat: enable themeing of opensource pages and components

### PR DESCRIPTION
**Pull request by jfcurran**
this pull-request is meant enable themeing which is described in #245 and should enable correct use of color on inteded pages (eg Kratos, Hydra, etc)

### CODE REVIEW

**Comment by jfcurran** [comment]
for reference, check out /oathkeeper /keto /hydra /kratos and /opensource on live and in this build preview. I think this is a welcome change 👍

**Comment by aeneasr** [comment]
Sweet!

File: src/styles/icons.css [hunk 1]
```diff
@@ -1,7 +1,15 @@
+i {
+  color: var(--colors-text-primary);
+}
+
 i.base-white {
   color: var(--colors-base-white);
 }
 
+i.text-priamry {
```
Comment (vinckr) [inline src/styles/icons.css:9]:
```suggestion
i.text-primary {
```

**Review by vinckr — COMMENTED** [review]
Looking great, love the colors, the new adopter component and the new content components 🎉 ❤️ 

Only thing I noticed now is that there is a gap in the adopter logos?
asd

Also, the code box still seems to change sizes a bit, but we could fix that in a later PR?

**Comment by aeneasr** [comment]
You have not tagged me for review yet, should i?

**Pull request merged**